app/workers/user_recommendations_worker.rb

```diff
@@ -0,0 +1,21 @@
+class UserRecommendationsWorker
+  include Sidekiq::Worker
+
+  def perform(user_id)
+    user = User.find user_id
+
+    recommendation = UserRecommendation.where(user_id: user.id).first
+
+    if recommendation.nil?
+      recommendation = UserRecommendation.create(user_id: user.id)
+    end
+
+    re = RecommendationsEngine.new
+
+    if recommendation.expire_at.nil? || recommendation.expire_at < Time.now
```

Comment:
@josepjaume I prefer the Redis approach you mentioned. Tomorrow I will refine this feature and it should be ready to be shippable (I'm not sure if that's a word XD)